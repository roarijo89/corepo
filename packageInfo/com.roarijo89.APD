{
	"name": "App editor",
	"desc_long": "ðŸš¨APD Is a version of DLG but we changed some things updated code and better UI You can use 3 fingers and tap 3 times on the screen and the APD menu will close or openðŸš¨<p><p><p>",
	"compatitle": "iOS 11 12 13 14",
	"changelog": "<strong>1.0</strong><br><br><strong>1.1</strong><br> added support to iOS 13.7 fixed some bugs<br>",
	"screenshot": null,
	"open": true,
}