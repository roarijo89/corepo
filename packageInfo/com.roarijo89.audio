{
	"name": "audiomack",
	"desc_long": "ðŸš¨premiumðŸš¨<p><p><p>",
	"compatitle": "iOS 11 12 13",
	"changelog": "<strong>1.0</strong><br>premium<br>",
	"screenshot": null,
	"open": true,

"warning": "Login info <br>Username:Coco <br> Password:cocox",


}