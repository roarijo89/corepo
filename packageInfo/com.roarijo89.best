{
	"name": "Best Fiends",
	"desc_long": "ðŸš¨in app purchase unlimited vip not testedðŸš¨<p><p><p>",
	"compatitle": "iOS 11 12 13",
	"changelog": "<strong>1.0</strong><br>in app purchase unlimited vip<br>",
	"screenshot": null,
	"open": true,

"warning": "Login info <br>Username:Coco <br> Password:cocox",

}