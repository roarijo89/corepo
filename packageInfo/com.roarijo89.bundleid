{
	"name": "Bundle ID finder app version",
	"desc_long": "ðŸš¨ Press on a app to copy bundle id ðŸš¨<p><p><p>",
	"compatitle": "iOS 11 12 13 14",
	"changelog": "<strong>1.0</strong><br><br>",
	"screenshot": null,
	"open": true,

}