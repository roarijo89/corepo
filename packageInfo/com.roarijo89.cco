{
	"name": "cco",
	"desc_long": "ðŸš¨ Flex to theos Coco Tweaks version   Use:cco on terminal or newtermðŸš¨<p><p><p>",
	"compatitle": "iOS 11 12 13 14",
	"changelog": "<strong>0.1</strong><br><br>",
	"screenshot": null,
	"open": true,

"warningt": "support <br>Flex 3<br> Flex 2 coming soon",

}