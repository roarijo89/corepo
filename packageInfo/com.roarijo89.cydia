{
	"name": "cydia 2021",
	"desc_long": "Cydia better UI and more<p><p><p>",
	"compatitle": "iOS 11 12 13 14",
	"changelog": "<strong>1.0</strong><br>2020<br>  <strong>1.1</strong><br>2021<br>",
	"screenshot": null,
	"open": true,
}