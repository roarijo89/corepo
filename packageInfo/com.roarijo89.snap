{
	"name": "Snapchat",
	"desc_long": "ðŸš¨no jailbreak detection canâ€™t tell the other person u took a screenshot prevents u from getting forced logoutðŸš¨<p><p><p>",
	"compatitle": "iOS 11 12 13",
	"changelog": "<strong>1.0</strong><br>added no jailbreak detection<br>added no screenshots notifications<br>added prevent ban /no force logout<br> <strong>1.2</strong><br>fixed the bug that when ur signed in to the cocotweaks login popup the whole app stops working<br>",
	"screenshot": null,
	"open": true,

"warning": "Donate <br>please donate to help us link in the popup",

}